docs/platforms/javascript/common/options.mdx

```diff
@@ -0,0 +1,454 @@
+---
+title: SDK Options
+description: "Learn more about how to configure the SDK. These options are set when the SDK is first initialized, passed to the init function as an object."
+customCanonicalTag: "/platforms/javascript/options/"
+sidebar_order: 2
+---
+
+<PlatformContent includePath="configuration/config-intro" />
+
+## Available Options
+
+<SdkOptionsOverview />
+
+## Core Options
+
+<SdkOption name='dsn' type='string' envVar='SENTRY_DSN'>
+The _DSN_ tells the SDK where to send the events. If this value is not provided, the SDK will try to read it from the `SENTRY_DSN` environment variable. If that variable also does not exist, the SDK will just not send any events.
```

Comment:
since we talked about env vars in the morning: We should change this description for client vs server